### DISCUSSION TITLE
BUILD_UTILS_SPAWN_SIGKILL: Command "npm run docs:build" exited with SIGKILL

### DISCUSSION THREAD

**Discussion by xinconan** [vercel/community#2134] [Help]
### Summary

I got build error with info: BUILD_UTILS_SPAWN_SIGKILL: Command "npm run docs:build" exited with SIGKILL.
In my local computer，It will build normal

**Comment by amyegan** [comment]
Hi @xinconan. Do you have any more error details in the [Build Logs](https://vercel.com/docs/concepts/deployments/logs#)? If you are able to also share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that would be very helpful for debugging.

**Reply by xinconan** [reply]
The full error:

```sh
error Command failed with signal "SIGKILL".
14:07:24.187 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
14:07:24.204 | Error: Command "yarn build" exited with 1
14:07:24.802 | Deployment completed
14:07:24.424 | BUILD_UTILS_SPAWN_1: Command "yarn build" exited with 1
```

Before the error, It was normal build console info。I used vitepress to deploy docs width 1000+ markdown files， may be not enough memory?
 I tried set memory use `node --max_old_space_size=8192 node_modules/vitepress/bin/vitepress.js build docs` but also got the same error.
